boot/freeldr/freeldr/ntldr/wlmemory.c

```diff
@@ -238,6 +239,60 @@ WinLdrSetupMemoryLayout(IN OUT PLOADER_PARAMETER_BLOCK LoaderBlock)
         return FALSE;
     }
 
+    // Always map acpi table if present
```

Comment:
You should probably mention the fact windows didn't boot unless you mapped it, that's good information to fill in here.

Reply:
The comment now is changed!